runtime/jitserver_launcher/module.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2019, 2019 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+<module xmlns:xi="http://www.w3.org/2001/XInclude">
+	<artifact type="executable" name="jitserver">
+		<include-if condition="spec.linux_390.*"/>
+		<include-if condition="spec.linux_ppc.*"/>
+		<include-if condition="spec.linux_x86.*"/>
+		<include-if condition="spec.flags.build_jitserver"/>
```

Comment:
This doesn't seem to work. Even when `build_jitserver` is not set in the spec, `jitserver` executable gets generated. @DanHeidinga @keithc-ca any idea why is that so?

Reply:
The `spec.linux` conditions should be redundant since you added the `build_jitserver` flag.

Are you adjusting the `build_jitserver` flag based on the configuration option, like [1]?

[1] https://github.com/ashu-mehra/openj9-openjdk-jdk11/blob/openj9/closed/OpenJ9.gmk#L357-L364

Reply:
> Are you adjusting the build_jitserver flag based on the configuration option,

Yes, created a PR for `OpenJ9.gmk` changes https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/350

Reply:
When I build with this and jdk8/350 and run `configure` without `--enable-jitserver`, no makefile is generated in `vm/jitserver_launcher`, signalling the desired behavior. After saying `make images`, there's no file called `jitserver` anywhere.

Reply:
Not sure what went wrong in my case. Anyway good if it is working. Thanks for trying it out.